src/modules/0.0.10/aws_iam/index.ts

```diff
@@ -35,7 +37,7 @@ export const AwsIamModule: Module2 = new Module2({
         // we are trusting aws won't change it from under us
         Object.is(a.assumeRolePolicyDocument, b.assumeRolePolicyDocument) &&
         Object.is(a.attachedPoliciesArns?.length, b.attachedPoliciesArns?.length) &&
-        a.attachedPoliciesArns?.every(as => !!b.attachedPoliciesArns?.find(bs => Object.is(as, bs))),
+        ((!a.attachedPoliciesArns && !b.attachedPoliciesArns) || !!a.attachedPoliciesArns?.every(as => !!b.attachedPoliciesArns?.find(bs => Object.is(as, bs)))),
```

Comment:
we should probably have a global util for comparing the equality two lists? We don’t check if `a` and `b` are undefined in other mappers

Reply:
It is true, what we do in other similar cases is default to `false`. I'll replicate that behaviour here.